RELEASING.md

```diff
@@ -0,0 +1,58 @@
+# Developer documentation
```

Comment:
This can be "Committer documentation" or something similar

Reply:
Nice catch, thanks